### PR TITLE
Add resource strings

### PR DESCRIPTION
**Pull request by chlowell**
@conniey #513 lacked 3 strings for `Microsoft.Fx.Portability.Reports.Html` (RazorEngine will throw without them).

### CODE REVIEW

**Comment by twsouthwick** [comment]
It would be nice to have a test that runs the report generator. If it's too much work, don't block on it, but could you create an issue for it if you can't add that?

**Comment by chlowell** [comment]
Turns out the test we have for this is broken. Fixing it now...

File: src/Microsoft.Fx.Portability.Reports.Html/Resources/LocalizedStrings.resx [hunk 1]
```diff
@@ -153,9 +153,15 @@
...
+    <value>PackageId</value>
+  </data>
   <data name="PortabilitySummaryPageTitle" xml:space="preserve">
     <value>Portability Summary</value>
   </data>
+  <data name="RecommendedChanges" xml:space="preserve">
+    <value>Recommended changes</value>
+  </data>
   <data name="RetargetingCompatIssueDescriptionPart1" xml:space="preserve">
     <value>Retargeting compatibility issues are breaking changes that only manifest when code is targeted to run on a newer .Net Framework version. This can happen if an application does not have a &lt;a href="https://msdn.microsoft.com/en-us/library/system.runtime.versioning.targetframeworkattribute%28v=vs.110%29.aspx"&gt;TargetFrameworkAttribute&lt;/a&gt; or when the application is rebuilt with a newer toolset.</value>
```
Comment (conniey) [inline src/Microsoft.Fx.Portability.Reports.Html/Resources/LocalizedStrings.resx]: The URL to TargetFrameworkAttribute should be: https://docs.microsoft.com/dotnet/api/system.runtime.versioning.targetframeworkattribute

**Comment by chlowell** [comment]
This isn't mergeable yet--it doesn't entirely resolve the Razor issues.

**Comment by chlowell** [comment]
@dotnet-bot test this please

**Comment by chlowell** [comment]
Per offline discussion, reworking the HTML report generation is sufficiently large and unrelated to warrant a separate PR. Merging the resource string additions alone for now.

**Pull request merged**